src/locales/en-US/commands/reddit.json

```diff
@@ -2,29 +2,29 @@
 	"name": "reddit",
 	"description": "Get information from Reddit",
 	"subredditName": "subreddit",
-	"subredditDescription": "Get a random post from a specified subreddit.",
+	"subredditDescription": "Get a random post from a specified subreddit",
 	"postName": "post",
-	"postDescription": "Get the information of a specific post.",
+	"postDescription": "Get the information of a specific post",
 	"optionsRedditName": "reddit",
-	"optionsRedditDescription": "The subreddit name to get a random post from. The `/r/` and `r/` prefixes are also supported.",
+	"optionsRedditDescription": "The subreddit name to get a random post from.",
```

Comment:
```suggestion
	"optionsRedditDescription": "The subreddit name to get a random post from",
```

Reply:
Why. That was there before too. You wrote that yourself.

Reply:
No, it wasn't:
![image](https://github.com/skyra-project/teryl/assets/24852502/888cb16b-65f4-461b-b29e-42cfd7147700)